src/fiat/bank-authorities.md

```diff
@@ -83,43 +76,114 @@ curl -X POST "https://service.centrapay.com/api/bank-authorities" \
   "modifiedBy": "crn:WIj211vFs9cNACwBb04vQw:api-key:MyApiKey",
 }
 ```
+
 **Error Responses**
 
-| Status |             Code              |                         Description                         |
-| :----- | :---------------------------- | :---------------------------------------------------------- |
-| 403    | {% break _ BANK_AUTHORITY_LIMIT_EXCEEDED %} | The account already has the max amount of bank authorities. |
+| Status |                           Code                           |                                                Description                                                 |
+| :----- | :------------------------------------------------------- | :--------------------------------------------------------------------------------------------------------- |
+| 403    | {% break _ BANK_AUTHORITY_LIMIT_EXCEEDED %}              | The account already has the max amount of bank authorities.                                                |
 | 403    | {% break _ BANK_AUTHORITIES_FOR_BANK_ACCOUNT_EXCEEDED %} | There are already two bank authorities for the provided bank account number, which is the maximum allowed. |
 
 ## Get information about a bank authority
 
 {% endpoint GET https://service.centrapay.com/api/bank-authorities/${id} %}
 
 ```sh
-curl -X GET https://service.centrapay.com/api/bank-authorities/Jaim1Cu1Q55uooxSens6yk \
+curl -X GET https://service.centrapay.com/api/bank-authorities/WRhAxxWpTKb5U7pXyxQjjY \
   -H "x-api-key: 1234"
 ```
 
 **Example response payload**
 
 ```json
 {
-    "id": "WRhAxxWpTKb5U7pXyxQjjY",
-    "accountId": "Jaim1Cu1Q55uooxSens6yk",
+  "id": "WRhAxxWpTKb5U7pXyxQjjY",
+  "accountId": "Jaim1Cu1Q55uooxSens6yk",
+  "bankAccountNumber": "12-1234-1234567-123",
+  "bankAccountName": "John Doe",
+  "status": "created",
+  "verified": false,
+  "directDebitAuthorized": true,
+  "createdAt": "2020-06-12T01:17:46.499Z",
+  "createdBy": "crn:WIj211vFs9cNACwBb04vQw:api-key:MyApiKey",
+  "modifiedAt": "2020-06-12T01:17:46.499Z",
+  "modifiedBy": "crn:WIj211vFs9cNACwBb04vQw:api-key:MyApiKey",
+}
+```
+
+<span id="finalize-anchor"></span>
+## Finalize information about a bank authority **EXPERIMENTAL**
+
+{% endpoint PUT https://service.centrapay.com/api/${accountId}/bank-authorities/${id}/finalize %}
+
+```sh
+curl -X PUT https://service.centrapay.com/api/accounts/Jaim1Cu1Q55uooxSens6yk/bank-authorities/WRhAxxWpTKb5U7pXyxQjjY/finalize \
```

Comment:
1. Finalize is too general... what if we need some other state change later?
2. Account id path param is redundant... bank authority id is globally unique
3. We follow [rest without put][1] as a design principle so usually we a posting to a sub resource.

What about `POST https://service.centrapay.com/api/bank-authorities/${id}/authorize-direct-debit`?

[1]: https://www.thoughtworks.com/radar/techniques/rest-without-put

Reply:
Agreed 💖, I've gone and updated this and pushed it up.